fabtests/benchmarks/rma_bw.c

```diff
@@ -81,6 +81,11 @@ static int run(void)
 			goto out;
 	}
 
+	if (fi->domain_attr->mr_mode & FI_MR_RAW) {
+		ret = fi_mr_unmap_key(domain, remote.key);
+		if (ret)
+			goto out;
+	}
```

Comment:
We don't want to update every test file to handle raw keys.  Can this be moved into a common area, similar to how the exchange is done?